src/workers/postUpdated.ts

```diff
@@ -182,22 +197,41 @@ const updatePost = async ({
 
   if (
     !databasePost ||
-    databasePost.metadataChangedAt.toISOString() >= updatedDate.toISOString()
+    databasePost.metadataChangedAt.toISOString() >=
+      data.metadataChangedAt.toISOString()
   ) {
     return null;
   }
 
   const title = data?.title || databasePost.title;
-  const updateBecameVisible = !databasePost.visible && !!title?.length;
+  const updateBecameVisible =
+    content_type === PostType.Freeform
+      ? databasePost.visible
+      : !databasePost.visible && !!title?.length;
 
   data.id = databasePost.id;
-  data.metadataChangedAt = updatedDate;
```

Comment:
Why did you remove this line?

Reply:
I moved it into the fixData function

Reply:
https://github.com/dailydotdev/daily-api/blob/d614e55bc016f47c98744117e877d52435f7283d/src/workers/postUpdated.ts#L341-L342